roles/flannel/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - { role: openshift_facts }
+- { role: etcd_client_certificates, etcd_generated_certs_dir: /etc/etcd/generated_certs, etcd_cert_prefix: node.etcd- }
```

Comment:
Any reason to use the node.etcd- prefix here instead of a certificate specific to flannel here? It would make it easier to have etcd reference a separate etcd instance once we have etcd sharding supported.